### PR TITLE
Removing Catherine maintainer mentions

### PR DESCRIPTION
**Pull request by CathPag**
### Describe your changes
Removing Catherine as maintainer mentions

### Checklist before opening this PR (put `x` in the checkboxes)
- [ ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by iamNoah1 — APPROVED** [review]

**Comment by iamNoah1** [comment]
@jihoon-seo @seokho-son don't we need to remove her also from the owners file(s)?

**Comment by nate-double-u** [comment]
> @jihoon-seo @seokho-son don't we need to remove her also from the owners file(s)?

I think so, but we can open a follow up issue.

**Review by nate-double-u — APPROVED** [review]
Thanks @CathPag

File: content/en/_index.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ and many more contributors.
...
 The Glossary is maintained by 
-[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), 
-[Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), 
-[Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/), 
 [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/), 
-and [Seokho Son](https://www.linkedin.com/in/seokho-son/).
+[Seokho Son](https://www.linkedin.com/in/seokho-son/),
+[Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/), 
+[Jorge Castro](https://www.linkedin.com/in/jorge-castro2112/),
+and [Nate W.](https://www.linkedin.com/in/nate-double-u/0.
```
Comment (nate-double-u) [inline content/en/_index.md:43]:
```suggestion
and [Nate W.](https://www.linkedin.com/in/nate-double-u/).
```

**Comment by nate-double-u** [comment]
It looks like https://github.com/cncf/glossary/pull/2343 takes care of the owners file.

**Pull request merged**